test_runner/gradle.properties

```diff
@@ -9,7 +9,8 @@
 
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-org.gradle.jvmargs=-Xmx2560m
+org.gradle.jvmargs=-Xmx2560m -Dfile.encoding=UTF-8
 kotlin.code.style=official
 org.gradle.parallel=true
-
+org.gradle.daemon=true
+org.gradle.caching=true
```

Comment:
```suggestion
org.gradle.caching=true

```

Reply:
i swear i have this setting on for new lines. I think it might only apply to knwn files